crawler/vison/spiders/visonSpider.py

```diff
@@ -1,16 +1,24 @@
 # -*- coding: utf-8 -*-
 import scrapy
 from scrapy.spiders import CrawlSpider
-import sqlite3
+import psycopg2
 
 class VisonspiderSpider(scrapy.Spider):
     name = 'visonSpider'
     start_urls = ['https://www.wikipedia.org/']
 
     def parse(self, response):
         # Cursor for the db
-        conn = sqlite3.connect('vison.db')
-        c = conn.cursor()
+        
+        conn = psycopg2.connect(
```

Comment:
Try handling the exception. What if the server is not connected?

Reply:
Sure will do so..